hatchet_sdk/loader.py

```diff
@@ -127,6 +136,28 @@ def get_config_value(key, env_var):
 
         tls_config = self._load_tls_config(config_data["tls"], host_port)
 
+        otel_exporter_oltp_endpoint = get_config_value(
+            "otel_exporter_oltp_endpoint", "HATCHET_CLIENT_OTEL_EXPORTER_OTLP_ENDPOINT"
+        )
+
+        otel_service_name = get_config_value(
+            "otel_service_name", "HATCHET_CLIENT_OTEL_SERVICE_NAME"
+        )
+
+        _oltp_headers = get_config_value(
+            "otel_exporter_oltp_headers", "HATCHET_CLIENT_OTEL_EXPORTER_OTLP_HEADERS"
+        )
+
+        if _oltp_headers:
+            otel_header_key, api_key = _oltp_headers.split("=", maxsplit=1)
+            otel_exporter_oltp_headers = {otel_header_key: api_key}
+        else:
+            otel_exporter_oltp_headers = None
```

Comment:
was considering error handling here - not sure what we usually do for things like this. We could catch a `ValueError` that would get thrown by `split` + the unpacking if there's no equal sign present. I think maybe that's a good strategy, and then can throw a helpful error about the headers being malformed. Any thoughts?